api/Makefile

```diff
@@ -11,24 +11,16 @@ RM_FILE      = rm -f
 INSTALL_DIR  = install -o root -g ${OSSEC_GROUP} -m 0750  -d
```

Comment:
```suggestion
INSTALL_DIR  = install -o root -g ${OSSEC_GROUP} -m 0750  -d
INSTALL_RW_DIR  = install -o root -g ${OSSEC_GROUP} -m 0770  -d
```

Reply:
Added in 2fc48a8612adcf3319996e315cd6fa801500548a.